timemachine/cpp/src/segmented_sumexp.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "device_buffer.hpp"
+
+namespace timemachine {
+
+// SegmentedSumExp is a class for computing the max and sum(exp(x - max) for x in xs) of a set of values
+// To compute the logsumexp you can use the code in kernels/k_logsumexp.cuh::compute_logsumexp_final given the
+// max and exp sum.
```

Comment:
Should this mention `SegmentedSumExp::logsumexp_host`?

Reply:
Added comment in https://github.com/proteneer/timemachine/pull/1239/commits/d6397fbc5d7a16b0f222222f220c676e03be0b9e